messages/data.bulk.results.md

```diff
@@ -1,22 +1,22 @@
 # summary
 
-Get the results of a Bulk API 2.0 ingest job.
+Get the results of a bulk ingest command that you previously ran.
 
 # description
 
-You can use this command to get successful/failed/unprocessed results of a bulk 2.0 ingest job written to different files in comma-separated values (CSV) format.
+Use this command to get the complete results after running one of the CLI commands that uses Bulk API 2.0 to ingest (import, upsert, or delete) large datasets to your org, such as "data import bulk". The previously-run bulk command must have completed; if it's still processing, run the corresponding resume command first, such as "data import resume." Make note of the job ID of the previous bulk command because you use it to run this command.
```

Comment:
```suggestion
Use this command to get the complete results after running one of the CLI commands that uses Bulk API 2.0 to ingest (import, update, upsert, or delete) large datasets to your org, such as "data import bulk". The previously-run bulk command must have completed; if it's still processing, run the corresponding resume command first, such as "data import resume." Make note of the job ID of the previous bulk command because you use it to run this command.
```